docs/user/document/publisher/contact_details-usage.en.md

```diff
@@ -1 +1,9 @@
 # Contact Details - Usage
+
+Lists means of contacting the publisher of the Advisories.
+
+At least one email address should be given for queries about the CSAF document.
+Email addresses of functions are preferred since people may change but the functions remain.
+If in doubt, a website with a contact form can also be given.
+For quick queries, a telephone number can also be provided.
+Listing postal addresses is unusual but possible.
```

Comment:
Add: "It can be useful, especially if a telephone number is given, as it helps to get an idea what timezone the team is located in.

Reply:
added